x-pack/legacy/plugins/apm/public/components/app/Main/route_config/utils.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { template, templateSettings } from 'lodash';
+import { routes } from './';
+
+export const getRouteByName = (routeName: string) =>
```

Comment:
What is this for? I think react-router provides this.

Reply:
This will return the route object filtering by the routeName.
ie: routeName: LINK_TO_TRACE
returnes: 
```
{
    exact: true,
    path: '/link-to/trace/:traceId',
    component: TraceLink,
    breadcrumb: null,
    name: RouteName.LINK_TO_TRACE
  }
```